src/main/java/org/jabref/gui/maintable/MainTableHeaderContextMenu.java

```diff
@@ -75,6 +80,11 @@ private void constructItems(MainTable mainTable) {
             RightClickMenuItem itemToAdd = createMenuItem(tableColumn, false);
             this.getItems().add(itemToAdd);
         }
+
+        this.getItems().add(separator);
+
+        RightClickMenuMoreItem itemToAdd = new RightClickMenuMoreItem("More");
```

Comment:
UI pattern: Use three dots if another menu optens:

```suggestion
        RightClickMenuMoreItem itemToAdd = new RightClickMenuMoreItem("More...");
```